public/app/features/datasources/DataSourceDashboards.test.tsx

```diff
@@ -19,17 +20,23 @@ const setup = (propOverrides?: object) => {
     loadPluginDashboards: jest.fn(),
     removeDashboard: jest.fn(),
     isLoading: false,
+    ...propOverrides,
   };
 
-  Object.assign(props, propOverrides);
-
-  return shallow(<DataSourceDashboards {...props} />);
+  return render(<DataSourceDashboards {...props} />);
 };
 
 describe('Render', () => {
+  it('should render without exploding', () => {
+    expect(() => setup()).not.toThrow();
+  });
   it('should render component', () => {
-    const wrapper = setup();
+    setup();
 
-    expect(wrapper).toMatchSnapshot();
+    expect(screen.getByRole('table')).toHaveClass('filter-table');
```

Comment:
```suggestion
    expect(screen.getByRole('table')).toBeInTheDocument();
```
I don't think testing for class name is necessary as it can change anytime.

Reply:
Good point, Thanks!